### PR TITLE
#2467 - add sgroup attachment point serializer/deserializer (mol v2000)

### PR DESCRIPTION
**Pull request by kaluginserg**
connected #2467

### NOTE
👉  If you wish to test these changes locally (with full understanding) - please use another branch (that contains changes from several connected PRs) #2768

### CODE REVIEW

**Comment by yuleicul** [comment]
Just for whom would like to check this PR, you can follow steps:

1. draw a functional group in Marvin, such as Bn, and remember its attachment point is the one with a **dashed** circle
 
    image

2. copy its lines of The Connection Table [CTAB] (V2000)
3. paste them in Ketcher
4. turn on `Show atom Ids` option in Ketcher
5. open the browser console, and check if the sgroup in `ketcher.editor.render.ctab.molecule.sgroups` has property `attachmentPoints` and attachment atom Ids are correct.

File: packages/ketcher-core/src/domain/serializers/mol/parseSGroup.js [hunk 1]
```diff
@@ -363,6 +365,42 @@ function identityMap(array) {
   return map
 }
 
+/**
+ * Superatom attachment point parsing for 'ctab' v2000
+ * @param ctabString {string} example '   1  1   2   0   '
+ *        M SAP sssnn6 iii ooo cc
```
[the remark this answers is not in the document]
Comment (kaluginserg) [reply packages/ketcher-core/src/domain/serializers/mol/parseSGroup.js:371]: I would add a link to GitHub issue #2467 (inside it we have the links to files and page, this link should be more stable than the external ones)

[the remark this answers is not in the document]
Comment (kaluginserg) [reply packages/ketcher-core/src/domain/serializers/mol/parseSGroup.js:371]: The link has been added to the jscomment.

**Comment by yuleicul** [comment]
I agree to approve this PR. In case merging it by accident, please notify me when it's ready to merge, and I'll approve it. @kaluginserg

**Comment by kaluginserg** [comment]
done

**Pull request closed without merging**